src/server/api/routes/voyages.router.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+// router setup
+
+const express = require("express");
+const router = express.Router({ mergeParams: true });
+const { authorizeUser } = require("../lib/middleware/auth.middleware");
+
+const ROLES = require("../../constants/roles");
+
+// controllers
+const voyagesController = require("../controllers/voyages.controller");
+
+// ENDPOINT: /api/voyages/ :GET
+router.get(
+  "/",
+  authorizeUser(ROLES.SUPER_USER, ROLES.ADMIN),
```

Comment:
This one should only be accessible to superusers.

Reason being that this is the endpoint to get all voyages in the entire application. It's really helpful to have, but it's not something that the operators or admins will ever have access to, since it would show data from outside their own organization. So this endpoint is mainly for debugging and maybe in the future to enable doing reports or something like that for Optimum Voyage. It's not for the end users.

End users will instead use the "Get voyages by vessel" endpoint.